### ISSUE TITLE
ScrollSnap/Screen Swipe

### ISSUE THREAD

**Issue by SimonDarksideJ** [Unity-UI-Extensions/com.unity.uiextensions#212]
<sup>Issue created by Former user as [Bitbucket Issue #&#8203;176](https://bitbucket.org/UnityUIExtensions/unity-ui-extensions/issues/176) on 2017.10.21 13:12.</sup>
Hey, I've been working on a script that is kind of a fork from your HorizontalScrollSnap script attempting to overcome some of the issues that I've been having with the HSS script. 

I have posted it on the Unity [forums](https://forum.unity.com/threads/screen-swipe-scroll-snap.501157/) but was hoping you could also check it out and give some feedback. I would really appreciate it. And of course if you like it feel free to include it in this library. The only thing that would need to change is the tweening stuff so its not relying on iTween.

**Comment by SimonDarksideJ** [comment]
<sup>On [2017.10.22 23:16](https://bitbucket.org/UnityUIExtensions/unity-ui-extensions/issues/176#comment-40701617), @SimonDarksideJ commented:</sup>
Thanks for the heads up. I'll be sure to look in to your script and take the changes into the project.

Could you list the changes you did or why the HSS wasn't completely up to scratch?  Was looking to implement an update to the control to address the fast swipe issues.

**Comment by SimonDarksideJ** [comment]
<sup>On [2017.10.23 01:29](https://bitbucket.org/UnityUIExtensions/unity-ui-extensions/issues/176#comment-40702321), Former user commented:</sup>
Hey I don't really know which changes I have made to the original HSS script. I build the script from scratch based off copying a pasting some of the drag events code from ScrollRect.cs to get the same feel. Then I implemented all the snapping stuff and velocity thresholds based on what I though felt best. I found it was easier that way than trying to understand exactly what was happening in HSS or ScrollSnapBase. Sorry I can't help you there.
